### PR TITLE
CONCF-561 acknowledge scrollbar on breakpoint small plus

### PR DESCRIPTION
**Pull request by bognix**
ping @Warkot @vforge

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :+1:

**Comment by rogatty** [comment]
LGTM

File: skins/oasis/css/core/breakpoints-variables.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@ $content-width-large: 1238px;
 $padding-width-large: 30px;
 $page-width-large: $content-width-large - 2 * $padding-width-large;
 // "special case" for 1024px tablets (think landscape iPad)
-$content-width-small-plus: 1024px;
+$content-width-small-plus: 1024px - $scroll-bar-width;
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-variables.scss]: what about this ocurrance:

Comment (vforge) [inline skins/oasis/css/core/breakpoints-variables.scss]: Fixed

**Comment by bognix** [comment]
LGTM

**Comment by bognix** [comment]
Tested. Tests for ads are no longer failing because of scrollbar.

**Pull request merged**